src/domain/individualCustomer/boatsCard/BoatCertificate.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Section from '../../../common/section/Section';
+import { BoatCert } from '../types';
+import styles from './boatsCard.module.scss';
+import Text from '../../../common/text/Text';
+import ExternalLink from '../../../common/externalLink/ExternalLink';
+import { formatDate } from '../../../common/utils/format';
+import { getFilename, getValidUntilColor } from './boatsCardUtils';
+
+export interface BoatCertificateProps {
+  certificate: BoatCert;
+}
+
+const BoatCertificate: React.FC<BoatCertificateProps> = ({ certificate }) => {
+  const { t, i18n } = useTranslation();
+  const {
+    file,
+    certificateType,
+    validUntil,
+    checkedAt,
+    checkedBy,
+  } = certificate;
+  const headerTextKey = `individualCustomer.customerBoats.${certificateType.toLocaleLowerCase()}`;
+  const filename = getFilename(file);
+
+  return (
+    <Section>
+      <div className={styles.boatCertificate}>
```

Comment:
Why don't we add the `className` directly here and get rid of this extra `div`?
```suggestion
    <Section className={styles.boatCertificate}>
```

Reply:
That is a good idea, but it seems that section is generating an `article` sub element so the `display: flex` from `styles.boatCertificate` is then applied to only that and not to the child elements which breaks the layout then.

Reply:
Fair enough!